site/_posts/2020-12-23-auth-facade-pattern.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: blog-post
+title: What is the auth facade pattern?
```

Comment:
Include something about upstream identity providers for more context. Maybe something like: 
"The Auth Facade: How to hide upstream identity providers"